recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
-{% set version = "1.2.1" %}
+{% set name = "clyent" %}
+{% set version = "1.2.2" %}
 
 package:
-  name: clyent
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: clyent-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/c/clyent/clyent-{{ version }}.tar.gz
-  md5: a8a953a636cbd75021c3983f7ca60b5f
+  git_url: https://github.com/Anaconda-Server/{{ name }}.git
+  git_tag: {{ version }}
```

Comment:
Right, but we could use an archive from GitHub (e.g. https://github.com/Anaconda-Platform/clyent/archive/1.2.2.tar.gz ).